<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>W_ALE_Config</class>
 <widget class="QDialog" name="W_ALE_Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UI配置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QComboBox QAbstractItemView::item { min-height: 26px;}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>UI配置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2">
      <property name="verticalSpacing">
       <number>9</number>
      </property>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBox_sizeMode"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>显示尺寸</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_MaskEnabled">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>网格背景</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>帧文本加粗</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkBox_TextBold">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>网格背景边线</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="checkBox_MaskBorderEnabled">
        <property name="text">
         <string>显示</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>W_ALE_Config</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
